cfgov/v1/jinja2/v1/story_page.html

```diff
@@ -0,0 +1,36 @@
+{% extends "layout-full.html" %}
+
+{% block hero %}
+    {% for block in page.header %}
+        {% if block.block_type == 'features' %}
+            <section class="wrapper
+                            wrapper__match-content
+                            block
+                            block__bg
+                            block__border
+                            block__flush-top
+                            u-mb0">
```

Comment:
Could this be `block__flush-bottom` instead?

Reply:
Let me try it. I directly copied the HTML from [campaign-page.html](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/v1/jinja2/v1/campaign_page.html) which is the only other page (aside from the homepage) that uses a jumbo hero.

Reply:
Hmmm yeah the utility class is used because `block__flush-bottom` removes the bottom border which we don't want.

<img width="722" alt="screenshot" src="https://user-images.githubusercontent.com/1060248/119722708-99a46300-be3a-11eb-8714-1783bf1dc4d9.png">